packages/core/src/Jobs/Products/Associations/Associate.php

```diff
@@ -73,7 +69,7 @@ public function handle()
                 $this->targets->map(function ($model) {
                     return [
                         'product_target_id' => $model->id,
-                        'type' => $this->type,
+                        'type' => $this->type->value,
```

Comment:
the propertu $type is nullable but its not cater
```suggestion
                        'type' => $this->type?->value,
```
but can it be null?

Reply:
maybe mix up with Dissociate job